logstash-core/lib/logstash/pipeline.rb

```diff
@@ -38,6 +38,7 @@ def self.validate_config(config_str, settings = {})
     begin
       # There should be a better way to test this
       self.new(config_str, settings)
+      true
     rescue => e
       e.message
```

Comment:
I'll document the method signature, is that enough? 
I believe the alternative implementation would be with just using an exception. Ruby doesn't have a good convention for return either true or an error message, as some other languages do (obviously erlang comes to mind :D )